### ISSUE TITLE
Newsletter from "Le Monde" parsing broken

### ISSUE THREAD

**Issue by desbma** [samuelclay/NewsBlur#1760]
I am using the "Email newsletter" feature, to forward a newsletter from "Le Monde" French news site to Newsblur.

The feed appears in Newsblur but the HTML parsing is completely broken, see below screenshot.

![Selection_2022-11-19_14-29-41](https://user-images.githubusercontent.com/1145175/202853145-7858fc13-6101-4e47-a691-0ba01790a12d.png)

I am using this feature for several other newsletter and this is the only one to be exhibit this behavior.

The exact HTML from the newsletter can be found here : https://asset.lemde.fr/newsletters/a-la-une/2022-11-19-6g9zxt8fu.html
This HTML properly displays in browser, or in a mail client (Thunderbird, GMail...) but not in Newsblur.

Thank you

**Comment by samuelclay** [comment]
I agree this should be fixed. I'll leave this ticket open until we can prioritize it, but having recently done this sort of work for another project, I think I have some ideas of how to improve parsing for emails.

**Comment by samuelclay** [comment]
Closing — since this Nov 2022 report, multiple newsletter parsing improvements have been shipped: charset/utf-8 header handling (fd1fa934a), HTML5 tags added to the scrubber (875e31926), better plain text handling (5efc134aa), and general improvements to the email-to-feed pipeline. If Le Monde newsletters are still broken, please open a new issue with the current error.
